components/automate-chef-io/layouts/_default/data-api.html

```diff
@@ -18,7 +18,7 @@
     </style>
   </head>
   <body>
-    <redoc spec-url="/api-docs/all-apis.swagger.json"></redoc>
+    <ReDoc spec-url="/api-docs/all-apis.swagger.json"></ReDoc>
```

Comment:
I felt like we might actually want to catch instances of all lowercase `redoc` and HTML is generally case insensitive, so this seemed like the least bad solution to this being a spellcheck failure.